docs/standard/garbage-collection/performance.md

```diff
@@ -633,20 +585,20 @@ This section describes the following procedures to isolate the cause of your per
           4      11       3       1               3
   ```
 
-  This information does not show when the garbage collection occurred, but you can determine the number of garbage collections that occurred in an interval of time. Assuming the worst case, the tenth generation 0 garbage collection finished at the start of the second interval, and the eleventh generation 0 garbage collection finished at the end of the fifth interval. The time between the end of the tenth and the end of the eleventh garbage collection is about 2 seconds, and the performance counter shows 3%, so the duration of the eleventh generation 0 garbage collection was (2 seconds * 3% = 60ms).
+  This information does not show when the garbage collection occurred, but you can determine the number of garbage collections that occurred in an interval of time. Assuming the worst case, the tenth generation 0 garbage collection finished at the start of the second interval, and the eleventh generation 0 garbage collection finished at the end of the fourth interval. The time between the end of the tenth and the end of the eleventh garbage collection is about 4 seconds, and the performance counter shows 3%, so the duration of the eleventh generation 0 garbage collection was (4 seconds * 3% = 120ms).
```

Comment:
@Maoni0 Could you check the changes I made for this example and the next one?

Reply:
Here's the preview of this section: https://review.docs.microsoft.com/en-us/dotnet/standard/garbage-collection/performance?branch=pr-en-us-30146#to-determine-the-length-of-time-in-a-garbage-collection

Reply:
I dunno how exactly it defines interval and etc so assuming interval 1 means second 1 and 2, and interval 2 means 3 and 4. and so on. this means the 11th GC finished somewhere in interval 3, ie, second 5 and 6, because this counter is only updated at the end of a GC.